init-tools.sh

```diff
@@ -18,41 +18,45 @@ __INIT_TOOLS_DONE_MARKER=$__PROJECT_JSON_PATH/done
 # Extended version of platform detection logic from dotnet/cli/scripts/obtain/dotnet-install.sh 16692fc
 get_current_linux_name() {
     # Detect Distro
-    if [ "$(cat /etc/os-release | grep -cim1 ubuntu)" -eq 1 ]; then
-        if [ "$(cat /etc/os-release | grep -cim1 16.04)" -eq 1 ]; then
+    if [ "$(cat /etc/*-release | grep -cim1 ubuntu)" -eq 1 ]; then
```

Comment:
Why have you changed these? We have switched from *-release to os-release about a month ago in https://github.com/dotnet/coreclr/pull/9263